sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -143,29 +141,9 @@ case class CreateDataSourceTableAsSelectCommand(
     mode: SaveMode,
     query: LogicalPlan,
```

Comment:
how do we guarantee this is resolved?

Reply:
It must be resolved before going to `CreateDataSourceTableAsSelectCommand`.

https://github.com/apache/spark/blob/030c1baf2bd13f392feceeb02fc6fed7a4c07b93/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala#L142-L144

Reply:
can we add an assert then?